site/_posts/2019-10-25-contour-v100rc2.md

```diff
@@ -53,4 +53,12 @@ This changes reduces the likelihood that Envoy can connect to a Contour instance
 
 ## Upgrading
 
-Please consult the [Upgrading]({% link _resources/upgrading.md %}) document for information on upgrading from Contour 1.0.0-rc.1 to Contour 1.0.0-rc.2.
+Please consult the [Upgrading][7] document for information on upgrading from Contour 1.0.0-rc.1 to Contour 1.0.0-rc.2.
+
+[1]: {{site.github.repository_url}}/releases/tag/v1.0.0-rc.2
+[2]: /getting-started
+[3]: /guides
+[4]: /docs
+[5]: /resources
+[6]: /docs/v1.0.0/httpproxy
+[7]: /resources/upgrading
```

Comment:
Can we use `{% link %}` here?

Reply:
For the top two, no. Link is a link to a based on the file provided. As /docs and /resources are directories it doesn't work.

I'll look at what I didn't use {% link %} for the bottom two